src/taxonomy/taxonomy-detail/TaxonomyDetailPage.jsx

```diff
@@ -74,6 +79,9 @@ const TaxonomyDetailPage = () => {
 
   return (
     <>
+      <Helmet>
+        <title>{getPageHeadTitle('', taxonomy.name)}</title>
```

Comment:
Feel free to push back on this, but what do you think about adding the Taxonomy header to give some context?

```suggestion
        <title>{getPageHeadTitle(taxonomy.name, intl.formatMessage(taxonomyMessages.headerTitle))}</title>

```

Reply:
Sure, done! Though I did it slightly differently. You always want the most specific information to come first in the `<title>` as that's what users can see most easily.